sotrplib/config/source_catalog.py

```diff
@@ -46,4 +46,21 @@ def to_source_catalog(self, log=None) -> SOCatFITSCatalog:
         )
 
 
-AllSourceCatalogConfigTypes = EmptySourceCatalogConfig | SOCatFITSCatalogConfig
+class SOCatWebskyCatalogConfig(SourceCatalogConfig):
+    catalog_type: Literal["websky"] = "websky"
+    path: Path | None = None
+    hdu: int = 1
+    flux_lower_limit: AstroPydanticQuantity = 0.03 * u.Jy
+
+    def to_source_catalog(self, log=None) -> SOCatWebskyCatalog:
+        return SOCatWebskyCatalog(
+            path=self.path,
+            hdu=self.hdu,
+            flux_lower_limit=self.flux_lower_limit,
+            log=log,
+        )
+
```

Comment:
Why is this a separate thing instead of just using a different socat file? Should this not be a separate PR too?